shortfin/README.md

```diff
@@ -167,16 +191,16 @@ recommended:
 * Enable LTO builds of libshortfin
 * Set flags to enable symbol versioning
 
-# Miscellaneous Build Topics
+## Miscellaneous build topics
 
-## Free-threaded Python
+### Free-threaded Python
 
 Support for free-threaded Python builds (aka. "nogil") is in progress. It
 is currently being tested via dev builds of CPython 3.13 with the
 `--disable-gil` option set. There are multiple ways to acquire such an
 environment. If using `pyenv`, here is a way:
 
-```
+```bash
 # Build a free-threaded 3.13 version.
 pyenv install --debug 3.13t-dev
```

Comment:
```suggestion
pyenv install 3.13t
```
We probably want to update this and go over the entire section as Python 3.13t was released in the meantime and there is no (strict) need to install a dev build.

Reply:
Good catch. Updated instructions for pyenv.